libs/barista-components/quick-filter/src/quick-filter-group.html

```diff
@@ -50,7 +50,9 @@
       }"
     ></ng-container>
 
-    <button class="dt-show-more" (click)="_showMore()" i18n>View more</button>
+    <button #viewMoreButton class="dt-show-more" (click)="_showMore()" i18n>
+      <span>View more</span>
```

Comment:
Why is this span necessary in the button now? Should not make any difference.

Reply:
Due to some formatting shenanigans, space is added around 'View more', a span is added here to ensure this doesn't happen